extensions/astra-viewer/neurodesk_astra_view/static/style.css

```
@@ -0,0 +1,54 @@
+.astra-viewer {
+  --astra-border: #d6dde5;
+  border: 1px solid var(--astra-border);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)